e2e/scripts/yum-install-firewalld.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+yum update -y
+yum install -y firewalld
```

Comment:
why not include this in the docker image(s)?

Reply:
updated